pkg/util/workqueue/prometheus/prometheus.go

```diff
@@ -31,7 +31,7 @@ const (
 	WorkQueueSubsystem         = "workqueue"
 	DepthKey                   = "depth"
 	AddsKey                    = "adds_total"
-	QueueLatencyKey            = "queue_latency_seconds"
+	QueueLatencyKey            = "queue_duration_seconds"
```

Comment:
I don't see deprecation for this?

Reply:
This metrics https://github.com/kubernetes/kubernetes/blob/c525d329effc6c6460cda947d1bf8092a927c2d3/pkg/util/workqueue/prometheus/prometheus.go#L69 is new introduced in 1.14 by https://github.com/kubernetes/kubernetes/pull/71300, to deprecate this metrics`queue_latency` https://github.com/kubernetes/kubernetes/blob/c525d329effc6c6460cda947d1bf8092a927c2d3/pkg/util/workqueue/prometheus/prometheus.go#L155

Reply:
What I think is the newly introduced metrics (via KEP, but name contains `latency/latencies`)  will not need to be replaced by another newer metrics (replace the name with `duration`), by this change, right ?

Reply:
Yes you’re totally right I just didn’t see it :)